go.mod

```diff
@@ -1,5 +1,7 @@
 module go.uber.org/sally
 
+go 1.14
```

Comment:
Just double checking if we want to specify the go version here and thereby change the checksum for go-bindata below

Reply:
Fair point. This probably isn't necessary and probably should be done separately.

Reply:
I'm seeing a checksum mismatch with 1.12 as well as 1.13.
I'm gonna guess go-bindata made this change.
I'll still omit the 1.14 change from this diff.